pkg/apiserver/registry/security/workloadidentity/storage/jwt_test.go

```diff
@@ -854,3 +881,29 @@ var _ = Describe("#TokenRequest", func() {
 		})
 	})
 })
+
+type fakeShootLister struct {
+	err error
+}
+type fakeShootNamespaceLister struct {
```

Comment:
Nit: I would put empty lines between the types. Also the functions at the bottom.

```suggestion
}

type fakeShootNamespaceLister struct {
```

Reply:
I address this with some other PR.
ref: https://github.com/gardener/gardener/pull/12357/